auth-web/public/config/configuration.json

```diff
@@ -1,14 +1,14 @@
 {
   "AUTH_URL": "https://dev.bcregistry.ca/cooperatives/auth/",
   "VUE_APP_PAY_ROOT_API": "https://pay-api-dev.pathfinder.gov.bc.ca/api/v1",
-  "VUE_APP_AUTH_ROOT_API": "https://auth-api-dev.pathfinder.gov.bc.ca/api/v1",	
+  "VUE_APP_AUTH_ROOT_API": "https://auth-api-dev.pathfinder.gov.bc.ca/api/v1",
   "VUE_APP_LEGAL_ROOT_API": "https://legal-api-dev.pathfinder.gov.bc.ca/api/v1",
   "VUE_APP_STATUS_ROOT_API": "https://status-api-dev.pathfinder.gov.bc.ca/api/v1",
   "VUE_APP_PATH_NEW_BUSINESS": "https://business-create-dev.pathfinder.gov.bc.ca/businesses/",
   "DIRECTOR_SEARCH_URL": "https://search-web-dev-1rdehl-dev.pathfinder.gov.bc.ca/",
   "VUE_APP_AUTH_RESET_API": "http://auth-api-dev.pathfinder.gov.bc.ca/test/reset",
   "LAUNCH_DARKLY_ENV_KEY": "5db9da115f58e008123cd783",
   "BCEID_OSD_LINK": "https://www.test.bceid.ca/os/?7686&SkipTo=Basic",
-  "FILE_SERVER_URL":"http://minio-1rdehl-dev.pathfinder.gov.bc.ca/public"
-
+  "FILE_SERVER_URL":"http://minio-1rdehl-dev.pathfinder.gov.bc.ca/public",
+  "NRO_URL": "https://dev.bcregistrynames.gov.bc.ca/nro/"
```

Comment:
Does this project have dc scripts for rebuilding the Openshift config? If so then this should be updated there as well.
And also, a note about adding this new configmap key needs to be in their release ticket.

Reply:
Yes, i believe Saravan has updated the Config Map but i'm not certain of the build scripts. Will follow up on this!

k, will do 👍